samples/sensor/accel_polling/README.rst

```diff
@@ -0,0 +1,38 @@
+.. _accel_polling:
+
+Generic 3-Axis accelerometer polling sample
```

Comment:
for bisectability: sample should be the last addition

Reply:
I reordered commits.